code/modules/clothing/under/_under.dm

```diff
@@ -30,7 +30,7 @@
 	. = ..()
 	if(random_sensor)
 		//make the sensor mode favor higher levels, except coords.
-		sensor_mode = pick(SENSOR_VITALS, SENSOR_VITALS, SENSOR_VITALS, SENSOR_LIVING, SENSOR_LIVING, SENSOR_COORDS, SENSOR_COORDS, SENSOR_OFF)
+		sensor_mode = pick(SENSOR_VITALS = 3, SENSOR_LIVING = 2, SENSOR_COORDS = 2, SENSOR_OFF = 1)
```

Comment:
Oh i forgot about this change 

```suggestion
		sensor_mode = pick_weight(SENSOR_VITALS = 3, SENSOR_LIVING = 2, SENSOR_COORDS = 2, SENSOR_OFF = 1)
```

Reply:
`pick_weight` doesn't work like that, you need to do `pick_weight(list(...)`.